app/controllers/articles_controller.rb

```diff
@@ -1,14 +1,24 @@
 class ArticlesController < ApplicationController
 
+  def getList
+    [{ value: "title",type: InputsHelper::INPUT_TEXT},{ value: "body",type: InputsHelper::INPUT_TEXT},
+      { value: "status",type: InputsHelper::INPUT_SELECT, select_list: ['public','private','archived']}]
+  end
+
+  helper_method :getList
+
   def index
     @user = current_user
-    @articles = @user.articles
-    @count = @articles.count
+    @articles = Article.where.not(user_id: @user.id)
+    @user_articles = @user.articles
+    @count = @user_articles.count
     @user_name = @user.username
   end
 
   def show
     @article = Article.find(params[:id])
+    @user = User.find(@article.user_id)
+    @days = ((Time.new - @article.created_at) / 86400).to_i
```

Comment:
@JuanMaGiraldo you can use helper methods to perform this kind of operation. What do the helpers do?